design-tokens/tokens/component.default.json

```diff
@@ -8154,6 +8166,18 @@
             }
           }
         },
+        "gapX": {
```

Comment:
<img width="802" alt="Screen Shot 2024-08-08 at 9 00 11 AM" src="https://github.com/user-attachments/assets/ff10ff96-a460-416f-809a-229294823580">

Looks like these are just added under `emphasized` kind -- but should be under `default` as well. Can you add the same tokens to those kinds as well?

Also seems like we're missing xlarge-6xl. There is a `component.xlarge...` token, so we can use that. For the larger size we don't have component[t-shirt size] for , just point to the `component.xlarge` token.